packages/reference/next.config.js

```diff
@@ -3,7 +3,9 @@ const withSass = require('@zeit/next-sass')
 const withCSS = require('@zeit/next-css')
 const nextRuntimeDotenv = require('next-runtime-dotenv')
 const withTM = require('next-transpile-modules')
-const withBundleAnalyzer = require('@zeit/next-bundle-analyzer')
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
+  enabled: process.env.ANALYZE === 'true'
```

Comment:
could we get a tighter name for this env var 'ANALYZE_BUNDLE' or something perhaps?

Reply:
oh I see elsewhere there's a BUNDLE_ANALYZE - perhaps we have more than one env var here